brave/src/main/java/com/linecorp/armeria/client/brave/BraveClient.java

```diff
@@ -57,109 +50,65 @@
 
     private static final Logger logger = LoggerFactory.getLogger(BraveClient.class);
 
-    // TODO(minwoox) Add the variant which takes HttpTracing.
-
-    /**
-     * Creates a new tracing {@link Client} decorator using the specified {@link Tracing} instance.
-     */
-    public static Function<Client<HttpRequest, HttpResponse>, BraveClient> newDecorator(Tracing tracing) {
```

Comment:
Just think if it's better to provide new implementation instead of providing both.